vault/base/route/route.yaml

```diff
@@ -6,6 +6,9 @@ metadata:
 spec:
   to:
     kind: Service
-    name: opf-vault-ui
+    name: opf-vault
+  tls:
+    termination: edge
+    insecureEdgeTerminationPolicy: Redirect
```

Comment:
@HumairAK General question about what's being done here: Is this `insecureEdgeTerminationPolicy: Redirect` connected to 

> redirectURIs:
          - https://vault-ui-vault.apps.smaug.na.operate-first.cloud/ui/vault/auth/oidc/oidc/callback
          - https://vault-ui-vault.apps.smaug.na.operate-first.cloud:8250/oidc/callback

If so, how does the route know to look at dex as for where to redirect to?

Reply:
@dystewart Yeah it's somewhat related. `insecureEdgeTerminationPolicy` redirects http traffic to https. This along with `edge` termination allows us vault to leverage smaug's api certs for a secure connection.